admin/database/database.go

```diff
@@ -276,7 +279,8 @@ type DB interface {
 	FindProjectInvite(ctx context.Context, projectID, userEmail string) (*ProjectInvite, error)
 	InsertProjectInvite(ctx context.Context, opts *InsertProjectInviteOptions) error
 	DeleteProjectInvite(ctx context.Context, id string) error
-	UpdateProjectInviteRole(ctx context.Context, id, roleID string) error
+	UpdateProjectInviteRole(ctx context.Context, id string, roleID string, resources []ResourceName) error
+	UpdateProjectInviteResources(ctx context.Context, id string, resources []ResourceName) error
```

Comment:
Can these be combined? Since you can already set resources in `UpdateProjectInviteRole`?